starter-code/js/index.js

```diff
@@ -1,13 +1,30 @@
-function deleteItem(e){
-
-}
+/*function deleteItem(e){
+  e.getElementByClassName("quantity").value --
+}*/
 
 function getPriceByProduct(itemNode){
-
+  var x = document.getElementById("price");
+    var text = "";
+    var i;
+    for (i = 0; i < x.length ;i++) {
+        text += x.elements[i].value + "<br>";
```

Comment:
If you put an `id` or class to the input that you are grabbing with `x.elements[i].value` will be more easier.